docs/reference/api/docker_remote_api_v1.24.md

```diff
@@ -4785,7 +4785,6 @@ image](#create-an-image) section for more details.
     Content-Type: application/json
 
     {
-      "Name": "top",
```

Comment:
`Name` is still part of the service update API - in fact, I believe that omitting it will trigger the service renaming check that you are added here. It is just imperative that the client not change the name from its previous value.